### PR TITLE
Update basic guide instructions

### PR DESCRIPTION
**Pull request by albertoxamin**
The current guide is still about the old version of the 3DBall example

### CODE REVIEW

**Comment by chriselion** [comment]
Thanks @albertoxamin - can you please sign the Contributor License Agreement?

File: docs/Basic-Guide.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ To set up the environment for training, you will need to specify which agents ar
...
+1. Each platform agent needs an assigned `Learning Brain`.  In this example, each platform agent was created using a prefab.  To update all of the brains in each platform agent at once, you only need to update the platform agent prefab.  In the **Project** window, go to the `Assets/ML-Agents/Examples/3DBall/Prefabs` folder. You should see the `3DBall` prefab in the **Inspector** window, now expand the prefab and select `Agent` which should appear now on your **Inspector** window.  In the **Project** window, drag the **3DBallLearning** Brain located in  `Assets/ML-Agents/Examples/3DBall/Brains` into the `Brain` property under `Ball 3D Agent (Script)` component in the **Inspector** window.
```
Comment (chriselion) [inline docs/Basic-Guide.md]: One nit: can you change "window, now" to "window. Now"?

Comment (albertoxamin) [reply docs/Basic-Guide.md]: Sure

**Comment by chriselion** [comment]
FWIW this conflicts with some changes on the develop branch. I'll merge this to master, then handle the conflicts on develop.

**Comment by chriselion** [comment]
Turns out this whole section got rewritten as part of a larger refactor, so I'm not going to try to merge it into the develop branch

**Pull request merged**